gtsfm/data_association/point3d_initializer.py

```diff
@@ -70,8 +84,12 @@ def execute_ransac_variant(self, track_2d: SfmTrack2d) -> np.ndarray:
             best_inliers: boolean array of length N. Indices of measurements
                are set to true if they correspond to the best RANSAC hypothesis
         """
+        if self.num_ransac_hypotheses is None:
+            raise ValueError("RANSAC triangulation requested but number of hypothesis is None.")
+
         # Generate all possible matches
-        measurement_pairs = self.generate_measurement_pairs(track_2d)
+        measurement_pairs = generate_measurement_pairs(track_2d)
+        random.shuffle(measurement_pairs)  # puts the random in RANSAC
```

Comment:
Line 267 has np.randomchoice, so it should already be random : - )

Reply:
Now it will be twice as random :)

Definitely missed that; I will change it back